test/lib/ansible_test/_util/controller/sanity/code-smell/runtime-metadata.py

```diff
@@ -202,6 +202,7 @@ def validate_metadata_file(path, is_ansible, check_deprecation_dates=False):
             ('deprecation'): Any(deprecation_schema),
             ('tombstone'): Any(tombstoning_schema),
             ('redirect'): fqcr,
+            ("action_plugin"): fqcr,
```

Comment:
This should only be valid for modules https://github.com/ansible/ansible/blob/devel/test/lib/ansible_test/_util/controller/sanity/code-smell/runtime-metadata.py#L236.

Reply:
Updated.